containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -589,6 +592,25 @@ public static AthenzJettyContainer createJettyContainer() {
         return container;
     }
 
+    public static void initConfigManager() {
+        System.getProperties().remove("socksProxyHost");
+
+        // First load initial properties (to get parameter store path)
+        String propFile = System.getProperty(AthenzConsts.ATHENZ_PROP_FILE_NAME,
+                getRootDir() + "/conf/athenz/athenz.properties");
+        ConfigProperties.loadProperties(propFile);
+
+        String awsParameterStorePath = System.getProperty(AthenzConsts.ATHENZ_PROP_AWS_PARAM_STORE_PATH, "");
```

Comment:
let's make the variable final and since we're checking for isEmpty below which handles nulls, there is no need to pass a default value